PokeApiNet/Models/Pokemon.cs

```diff
@@ -1111,6 +1122,25 @@ public class PokemonSpecies : NamedApiResource
         public List<PokemonSpeciesVariety> Varieties { get; set; }
     }
 
+    public class PokemonSpeciesFlavorTexts
+    {
+        /// <summary>
+        /// The localized flavor text for an api resource in a specific language
+        /// </summary>
+        [JsonProperty("flavor_text")]
+        public string FlavorText { get; set; }
+
+        /// <summary>
+        /// The version.
```

Comment:
I try to match the property docs to the docs listed on the PokeAPI docs site where it makes sense. [See here](https://pokeapi.co/docs/v2.html#flavortext)

```suggestion
        /// The game version this flavor text is extracted from.
```